.github/workflows/ci.yml

```diff
@@ -47,28 +43,22 @@ jobs:
       image: 'erlang:${{ matrix.otp_vsn }}'
     env:
       LATEST_OTP_RELEASE: 23.1
+      RUNNING_ON_CI: true
     steps:
-      - shell: bash
-        run: |
-          # upgrade git to 2.18+ for actions/checkout@v2
-          set -euo pipefail
-          set -x
-          # some erlang containers are based on jesse, some or stretch, etc
-          # NOTE jesse has no VERSION_CODENAME in /etc/os-release
-          # NOTE lsb_release is not installed
-          # see https://unix.stackexchange.com/questions/180776/how-to-get-debian-codename-without-lsb-release
-          VERSION_CODENAME=$(dpkg --status tzdata|grep Provides|cut -f2 -d'-')
-          VCB=${VERSION_CODENAME}-backports
-          echo "deb http://cloudfront.debian.net/debian ${VCB} main" >>/etc/apt/sources.list.d/${VCB}.list
-          echo "deb-src http://cloudfront.debian.net/debian ${VCB} main" >>/etc/apt/sources.list.d/${VCB}.list
-          apt-get update -y --fix-missing -o Acquire::Check-Valid-Until=false
-          apt-get -t ${VCB} install -y git
+      - run: |
+          apt-get update
+          apt-get -y install tcl tcl-dev gettext
+          wget https://github.com/git/git/archive/v2.18.0.tar.gz -O git.tar.gz
```

Comment:
```suggestion
          cd /usr/src/
          wget https://github.com/git/git/archive/v2.18.0.tar.gz -O git.tar.gz
```

Reply:
Why is this required? I can accept it, I just don't understand the goal here. One could even argue that the result of `mktemp` would be a better place to have it.

Reply:
`/usr/src` holds sources. Like you say you could also use a temporary folder, but on a non-CI machine, you usually unpack *sources* in `/usr/src`. The more important goal is not to voluntarily pollute the working directory.

Reply:
Sure, I can work with that.